### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by luojie233**
Signed-off-by: Jie Luo luo612@zju.edu.cn

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2027)

### CODE REVIEW

File: docs/user-guide/jobs.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ parallelism, for a variety or reasons:
 - If the controller failed to create pods for any reason (lack of ResourceQuota, lack of permission, etc.),
   then there may be fewer pods than requested.
 - The controller may throttle new pod creation due to excessive previous pod failures in the same Job.
-- When a pod is gracefully shutdown, it make take time to stop.
+- When a pod is gracefully shutdown, it makes take time to stop.
```
Comment (caesarxuchao) [inline docs/user-guide/jobs.md]: s/makes take/takes

File: docs/user-guide/kubectl/kubectl_drain.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Drain node in preparation for maintenance
...
+The given node will be marked unschedulable to prevent new pods from arriving. 'drain' evicts the pods if the APIServer supports evicition (http://kubernetes.io/docs/admin/disruptions/). Otherwise, it will use normal DELETE to delete the pods. The 'drain' evicts or deletes all pods except mirror pods (which cannot be deleted through the API server).  If there are DaemonSet-managed pods, drain will not proceed without --ignore-daemonsets, and regardless it will not delete any DaemonSet-managed pods, because those pods would be immediately replaced by the DaemonSet controller, which ignores unschedulable markings.  If there are any pods that are neither mirror pods nor managed by ReplicationController, ReplicaSet, DaemonSet, StatefulSet or Job, then drain will not delete any pods unless you use --force.
```
Comment (caesarxuchao) [inline docs/user-guide/kubectl/kubectl_drain.md]: What's changed here? Github is confused :P

**Comment by caesarxuchao** [comment]
One nit and one question, otherwise lgtm. Please squash :)

File: docs/user-guide/jobs.md (hunk 1 above)
Comment (luojie233) [inline docs/user-guide/jobs.md]: OK

File: docs/user-guide/kubectl/kubectl_drain.md (hunk 2 above)
Comment (luojie233) [inline docs/user-guide/kubectl/kubectl_drain.md]: changed the word "evicition" before the link.

**Comment by luojie233** [comment]
I changed and squashed them. @caesarxuchao

File: docs/user-guide/kubectl/kubectl_drain.md (hunk 2 above)
Comment (caesarxuchao) [inline docs/user-guide/kubectl/kubectl_drain.md]: It's still evicition after the change, could you fix it?

Comment (luojie233) [inline docs/user-guide/kubectl/kubectl_drain.md]: eviciton-->evicition, I don't know why there is no highlight.

Comment (caesarxuchao) [inline docs/user-guide/kubectl/kubectl_drain.md]: Should be `eviction`, not `evicition`.

**Comment by caesarxuchao** [comment]
LGTM. Thanks. @devin-donnelly can you merge it? Thanks.

**Pull request merged**